libs/ui/elements/src/lib/thumbnail/thumbnail.component.ts

```diff
@@ -43,6 +51,10 @@ export class ThumbnailComponent implements AfterViewInit, OnDestroy {
     private changeDetector: ChangeDetectorRef
   ) {}
 
+  ngOnInit() {
+    this.setPlaceholderObjectFit()
+  }
```

Comment:
I don't think it's a good idea to rely on ngInit for something that is essentially a computed value

Maybe you could do a getter, like `get objectFit_() { return this.isPlaceholder ? 'scale-down' : this.fit }`?

Reply:
Ah, great hint, I was just thinking in `setters` and did not want to implement two because of the uncertainty which one would be executed first. The `getter` is the solution :-)